compiler/ssa.go

```diff
@@ -442,12 +442,12 @@ func (c *SSAFunctionCompiler) Compile(importTypeIDs []int) {
 				c.PushStack(targetValueID)
 			}
 
-		case "current_memory":
+		case "memory.size":
```

Comment:
We could avoid the strings in this switch if upstream merges: https://github.com/go-interpreter/wagon/pull/172

Reply:
Agreed; I left a suggestion comment over in https://github.com/go-interpreter/wagon/pull/172. It's not absolutely mandatory, though regardless of whichever option is chosen for converting an opcode into a string, we should definitely incorporate the change int a separate PR.